File: 250720/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.py
fruits = ['apple', 'banana', 'grape', 'bluberry', 'orange']
char = input()
cnt = 0
for fruit in fruits:
    if len(fruit) > 3 and (fruit[2] == char or fruit[3] == char):
        print(fruit)
        cnt += 1
print(cnt)

File: codetree/260402/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);
        
        String[] arr = {"apple", "banana", "grape", "blueberry", "orange"};   
        char input = sc.next().charAt(0);
        int cnt = 0;

        for (int i = 0; i < 5; i++) {
            if (input == arr[i].charAt(2) || input == arr[i].charAt(3)){
                System.out.println(arr[i]);
                cnt++;
            }
        }

        System.out.print(cnt);
    }
}